src/main/java/uk/co/evoco/webdriver/configuration/TolerantActionExceptions.java

```diff
@@ -0,0 +1,46 @@
+package uk.co.evoco.webdriver.configuration;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+import java.util.List;
+
+public class TolerantActionExceptions {
+
+    private String waitTimeoutInSeconds;
+    private List<String> exceptionsToHandle;
+
+    /**
+     *
+     * @return the tolerant action wait time in seconds
+     */
+    String getWaitTimeoutInSeconds() {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use explicit scoping instead of the default package private level](https://app.codacy.com/gh/digital-delivery-academy/selenium-pom-framework/pullRequest?prid=5208619)